frontend/plan/src/components/CartSection.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import "../styles/course-cart.css";
+
+class CartSection extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    toggleCheck = () => {
+        this.props.toggleCheck();
+    };
+
+    render() {
+        const checkStyle = {
+            width: "1rem",
+            height: "1rem",
+            borderRadius: "1rem",
+            backgroundColor: "white",
+            border: "1px solid grey",
+        };
+        return (
```

Comment:
Maybe split these up into their own components? They can still be in this file but it'll make it easier to read the render function for this component.